internal/lpastore/lpa.go

```diff
@@ -107,7 +107,11 @@ func CreateLpaFromDonorProvided(donor *donordata.Provided) CreateLpa {
 		body.HowReplacementAttorneysStepInDetails = donor.HowShouldReplacementAttorneysStepInDetails
 	}
 
-	if donor.ReplacementAttorneys.Len() > 1 && (donor.HowShouldReplacementAttorneysStepIn.IsWhenAllCanNoLongerAct() || !donor.AttorneyDecisions.How.IsJointlyAndSeverally()) {
+	needsDecisions := donor.ReplacementAttorneys.Len() > 1 && (donor.Attorneys.Len() == 1 ||
```

Comment:
once this is merged then this can be too https://github.com/ministryofjustice/opg-data-lpa-store/pull/347